src/server/utilities/liteATITracking/viewTracking/index.ts

```diff
@@ -0,0 +1,24 @@
+export default async () => {
+  if (typeof window.IntersectionObserver === 'undefined') {
+    console.log('OBSERVER POLYFILL ME OUT!!!!');
+    // Polyfill IntersectionObserver, e.g. for IE11
+    await import('intersection-observer');
+  }
+
+  const observer = new IntersectionObserver(entries => {
+    console.log('CHECK ME OUT!!!!');
+    entries.forEach(entry => {
+      if (entry.isIntersecting) {
```

Comment:
Do we need to apply similar rules to the canonical view tracker e.g. 50% in view? https://github.com/bbc/simorgh/blob/9763a3cf74f76ad41801ff6b5836343c201d8eba/src/app/hooks/useViewTracker/index.jsx#L58-L63